lib/guest_installation_and_configuration_base.pm

```diff
@@ -772,7 +772,15 @@ sub config_guest_network_bridge_device {
         if ($self->{guest_netaddr} ne 'host-default') {
             $self->write_guest_network_bridge_device_config($_bridge_network, $_bridge_device, 'static', 'auto', '', 'yes');
             my $_bridge_device_config_file = '/etc/sysconfig/network/ifcfg-' . $_bridge_device;
-            script_retry("wicked ifup $_bridge_device_config_file $_bridge_device", retry => 3, die => 0);
+            if (is_opensuse) {
+                #NIC in openSUSE TW guest is unable to get the IP from its network configration file with 'wicked ifup' or 'ifup'
+                #Not sure if it is a bug yet. This is just a temporary solution.
+                my $_bridge_ipaddr = script_output("grep IPADDR $_bridge_device_config_file | cut -d \"'\" -f2");
+                script_retry("brctl addbr $_bridge_device; ip addr add $_bridge_ipaddr dev $_bridge_device; ip link set $_bridge_device up", retry => 3, die => 0);
```

Comment:
I recommend to combine the commands with `&&`, not `;` so that the command would fail and not hide errors on any error in one of the commands